src/components/Pagination.tsx

```diff
@@ -0,0 +1,79 @@
+import { Dispatch, SetStateAction, useMemo } from "react";
+import { IconButton } from "src/components";
+import { Css, Palette } from "src/Css";
+import { SelectField } from "src/inputs";
+import { useTestIds } from "src/utils";
+
+export type PageSettings = {
+  page: number;
+  perPage: number;
+};
+
+type PaginationProps = {
+  label: string;
+  totalCount: number;
+  settings: PageSettings;
+  setSettings: Dispatch<SetStateAction<PageSettings>>;
+};
+
+// Make a list of 50/100/150/etc page sizes for the user to chose
+export const pageOptions = Array(5)
+  .fill(0)
+  .map((_, i) => ({ id: (i + 1) * 50, name: ((i + 1) * 50).toString() }));
```

Comment:
**request:** Per the [component style guide](https://homebound.atlassian.net/wiki/spaces/ENG/pages/339574863/Component+Style+Guide), can we move this const to the bottom of the component?

Reply:
cool, really nice this style guide! Thanks for share 👍